app/db/db_functions.py

```diff
@@ -0,0 +1,34 @@
+from app.db.oracle import conn
+
+
+def get_results(result, label=""):
```

Comment:
This should definitely have a comment. This is used specifically for the style of data that Derek sends us through his procs. The column value of "program_code:2-BU-TEST" can be turned into a dict.